static/js/lib/learning_resources.js

```diff
@@ -143,16 +147,20 @@ export const resourceLabel = (resource: string) => {
     : concat(capitalize(resource), "s")
 }
 
-export const maxPrice = (courseRun: ?CourseRun) => {
-  const prices = courseRun && courseRun.prices ? courseRun.prices : []
+export const maxPrice = (prices: Array<CoursePrice>) => {
+  if (emptyOrNil(prices)) {
+    return null
+  }
   const price = Math.max(...prices.map(price => price.price))
-  return price > 0 ? `$${price}` : "Free"
+  return price > 0 ? `$${price.toFixed(2)}` : "Free"
```

Comment:
Can we make a separate function for this? Maybe similar to https://github.com/mitodl/mitxpro/blob/master/static/js/lib/ecommerce.js#L39

Reply:
Done